edx_notifications/startup.py

```diff
@@ -7,10 +7,8 @@
 
 from edx_notifications.timer import register_purge_notifications_timer  # pylint: disable=unused-import
 from edx_notifications.scopes import SingleUserScopeResolver, register_user_scope_resolver
-from edx_notifications.digests import (  # pylint: disable=unused-import
-    register_digest_timers,
-    create_default_notification_preferences
-)
+from edx_notifications.digests import create_default_notification_preferences
+from edx_notifications.digests import register_digest_timers  # pylint: disable=unused-import
```

Comment:
@nasirhjafri Is there a reason to split edx_notifications.digests imports in two separate imports?

Reply:
isort did it. Should I revert it back?

Reply:
in that case you can use isort:skip here to avoid multiline import.